src/views/LoginPage.tsx

```diff
@@ -1,81 +1,57 @@
 import React from 'react';
-import {useForm} from 'react-hook-form';
-import * as yup from 'yup';
-import {yupResolver} from '@hookform/resolvers/yup';
-import {Link} from "react-router-dom";
-import {Logo} from "../assets";
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Link } from 'react-router-dom';
+import { loggingSchema } from '../components/yupSchemas/LoggingSchema';
+import { Logo } from '../assets';
+import { DefaultInput } from '../components/ComponentParts/DefaultInput';
+import { RedButton } from '../components/ComponentParts/RedButton';
 
 interface LoginCredentials {
-  email: string,
-  password: string,
+  email: string;
+  password: string;
 }
 
 export const LoginPage: React.FC = () => {
-
-  const schema = yup.object().shape({
-    email: yup
-      .string()
-      .email()
-      .required("To pole jest wymagane."),
-    password: yup
-      .string()
-      .required("To pole jest wymagane."),
-  })
-
-  const {register, handleSubmit, reset, formState: {errors}} = useForm<LoginCredentials>({
-    resolver: yupResolver(schema),
+  const {
+    control,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<LoginCredentials>({
+    resolver: yupResolver(loggingSchema),
   });
 
-  const onSubmit = ((formValues: LoginCredentials) => {
+  const onSubmit = (formValues: LoginCredentials) => {
     console.log('form data is', formValues);
     reset({
-      email: "",
-      password: ""
+      email: '',
```

Comment:
Ktoś tu ma problemy z prettierem: @Annshka to chyba u Ciebie :)

Reply:
Też tego nie zauważyłem. Nie ma problemu, zaraz poprawię.

Reply:
Ty masz dobrze, po prostu sformatowałeś kod i to się zrobiło:-) to @Annshka musi spojrzeć, czemu nie formatuje się jej zgodnie z plikiem konfiguracyjnym prettiera